tests/test_resolver.py

```
@@ -15,86 +14,85 @@ def resolver():
-    assert resp[0]['host'] == '127.0.0.2'
+    assert await resolver.resolve("test.com") == "127.0.0.1"
+    resp = await resolver.resolve("test2.com")
+    assert resp[0]["host"] == "127.0.0.2"
     assert resolver._resolve.call_count == 2
 
-    with mocker.patch('aiodns.DNSResolver.query', side_effect=f), pytest.raises(
-        Exception
-    ):
-        await resolver.resolve('test3.com')
+    mocker.patch("aiodns.DNSResolver.query", side_effect=f)
+    with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)